composer.json

```
@@ -16,7 +16,7 @@
     "require": {
         "composer-plugin-api": "^1.1 || ^2.0",
         "composer/composer": "^1.10.23@stable || ^2.1.9",
-        "drush/drush": "^9.0|^10.0",
+        "drush/drush": "^9.0|^11.0",
```

This change removes the support of Drush 10, but Scripthor still supports Drush 10.

```suggestion
        "drush/drush": "^9.0|^10.0|^11.0",
```